File: examples/google/search/go/searchForVideos.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/google"
)

// Search for videos on Google
func SearchForVideos() {
	googleService := google.NewGoogleService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := googleService.Search(&google.SearchRequest{
		Query: "how to make donuts",
	})
	fmt.Println(rsp, err)
}

File: examples/js/google/search/searchForVideos.js
const { GoogleService } = require("m3o/google");

// Search for videos on Google
async function searchForVideos() {
  let googleService = new GoogleService(process.env.MICRO_API_TOKEN);
  let rsp = await googleService.search({
    query: "how to make donuts",
  });
  console.log(rsp);
}

searchForVideos();
